src/ape/contracts/base.py

```diff
@@ -1,16 +1,15 @@
-from typing import TYPE_CHECKING, Any, Dict, List, Optional, Union
+from typing import TYPE_CHECKING, Any, Dict, Iterator, List, Optional, Tuple, Union
 
```

Comment:
At some point, the `contract` module can unbundle a bit if it grew too large, there's only one module in there